app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
```

Comment:
có lỗi CI khi chạy reek: User#remember_token is a writable attribute (Attribute), nên em có sửa bằng cách ignore ở file .reek, a có cách sửa nào khác thì bảo em với 
![screenshot from 2017-07-20 11-21-55](https://user-images.githubusercontent.com/20549970/28400524-e014a388-6d3d-11e7-8c9f-6dc1731eebfd.png)